UpdateMunicipalities/function.json

```diff
@@ -0,0 +1,11 @@
+{
+  "bindings": [
+    {
+      "schedule": "0 */15 * * * *",
```

Comment:
how many times is this supposed to run ? once a day suffices

Reply:
Which hour?

Reply:
I've set every day at 01:00 AM